rest/server_context.go

```diff
@@ -46,6 +46,8 @@ type ServerContext struct {
 	statsContext *statsContext
 	HTTPClient   *http.Client
 	replicator   *base.Replicator
+	mutexRunning base.AtomicBool
+	blockRunning base.AtomicBool
```

Comment:
These would be cleaner as just global variables, and renamed to mutexProfileRunning/blockProfileRunning for clarity.

Reply:
Done